GEN-1137: Add neo4j block storage format notice (#42)
Linkurious/documentation-resources: Customer Support Procedures/Neo4j-v5.0-migration.md

to fully reindex your data-source.

See complete procedure on [Neo4j documentation](https://neo4j.com/docs/upgrade-migration-guide/current/version-5).

# New block storage format

> To improve database performances, Neo4j has implemented a new storage format (so called `block`)

Reviewer: "performance" is always singular in English

```suggestion
To improve database performance, Neo4j has implemented a new storage format (so called `block`)
```

in v5.14 and made it the default setting for the new databases created in the Enterprise Edition since v5.17.

> [!NOTE]
> The storage format for existing databases is not automatically migrated to the new `block` format,
> it requires a manual intervention.

> [!CAUTION]
> The `block` format is not available on the Neo4j Community Edition (CE).

> > You can be impacted at any time in the feature if you plan an upgrade of an existing database

Reviewer: suggestion for simplicity and clarity
```suggestion
> You can be impacted if you plan to upgrade an existing database
```

> from the Neo4j Community Edition (CE) to the Neo4j Enterprise Edition (EE).

> Despite the new format doesn't imply any side effect to Linkurious Enterprise (when used on a new db),
> the actual migration from an old format to the `block` format (on an existing db) will cause a change to the Neo4j internal IDs of the nodes and relationships ([source](https://neo4j.com/docs/operations-manual/current/database-internals/store-formats/#change-store-format)).

Reviewer: suggestion for simplicity and clarity
```suggestion
The new `block` format is compatible with Linkurious Enterprise, but migrating an existing database to
`block` format will cause changes to the Neo4j internal IDs of the nodes and relationships ([source](https://neo4j.com/docs/operations-manual/current/database-internals/store-formats/#change-store-format)).
```

> [!CAUTION]
> If the references (Neo4j internal IDs) used by Linkurious Enterprise to access data change,

> > the coherence of the visualizations and cases stored in Linkurious Enterprise is compromised
> > with a very high risk of seeing different information than what was originally saved.

Reviewer: - [coherence](https://www.wordreference.com/fren/coherence) is frequent in French but rare in English, suggesting a more commonly used on 
- clarifying the impact in more concrete terms

```suggestion
> the integrity of visualizations and cases stored in Linkurious Enterprise will be compromised,
> causing the wrong nodes and relationships to be displayed as the content of these visualizations and cases.
```

If you are already using [Alternative IDs](https://doc.linkurious.com/admin-manual/latest/alternative-ids/) in Linkurious Enterprise,
you are not relying anymore on the Neo4j internal IDs and hence not impacted by this section.
In this configuration you can safely perform the migration.

If you are **NOT** using [Alternative IDs](https://doc.linkurious.com/admin-manual/latest/alternative-ids/) in Linkurious Enterprise
**AND** you are planning to migrate to the `block` format

> **AND** you don't want to lose consistency of your data (e.g. it might not be a requirement for DEV environments),

Reviewer: clarifying the option here: concretely it means being ok with deleting al viz and cases
```suggestion
**AND** you do not want to delete all existing visualizations and cases in Linkurious Enterprise,
```
